packages/utilities/src/BaseComponent.test.tsx

```diff
@@ -47,6 +47,7 @@ class TestComponent extends BaseComponent<{}, {}> {
   }
 
   private _createNullRef(): void {
+    // tslint:disable-next-line:no-any
     let foo: any = null;
```

Comment:
`Function` instead of `any`?

Reply:
`null` is not assignable to type `Function`.  'Function | null` doesn't work because `foo()` can't be called since it might be null. Figured I'd leave it as `any` because the unit test is testing for the error